### ISSUE TITLE
drop go dependency ?

### ISSUE THREAD

**Issue by cmdoret** [ABignaud/transcription_T7_analysis#1]
Hi @ABignaud, I noticed you are using dnaglider in your smk workflow :grin: 
Unfortunately I did not make it into a conda package, which is a shame because Go is only required to compile it (and not run it).

As I don't have time to make a go pkg now, maybe a more convenient alternative for you would be [seqkit](https://anaconda.org/bioconda/seqkit), another go package which is on bioconda.

Here:

https://github.com/ABignaud/transcription_T7_analysis/blob/8de68b1f431e42454c9bca003967d3fd2d77ee18/rules/04_ecoli_matrix_analysis.smk#L15-L22

You could have something like this instead:

```sh
seqkit sliding --step 100 --window 500 W303_SGD_2015_JRIU00000000.fa.gz | seqkit fx2tab --gc | cut -f1,4

gi|696447993|gb|JRIU01000414.1|_sliding:901-1400	38.80
gi|696447993|gb|JRIU01000414.1|_sliding:1001-1500	38.80
gi|696447993|gb|JRIU01000414.1|_sliding:1101-1600	38.60
gi|696447993|gb|JRIU01000414.1|_sliding:1201-1700	38.60
gi|696447993|gb|JRIU01000414.1|_sliding:1301-1800	38.80
gi|696447993|gb|JRIU01000414.1|_sliding:1401-1900	39.80
gi|696447993|gb|JRIU01000414.1|_sliding:1501-2000	40.00
gi|696447993|gb|JRIU01000414.1|_sliding:1601-2100	39.80
```

This would allow you to just add seqkit to the conda env and drop go from dependencies

**Comment by ABignaud** [comment]
Yes I agree it would have been better to avoid the dependency issues. I just start to do the workflow to compile the analysis pipelines I used to generates the figures for the paper. I used the output of dnaglider to generate the GC file (outside the workflow at the beginning) used in the analysis so that's why I used it in the rule even if the dependency is painful.

**Comment by cmdoret** [comment]
Makes sense, I added an issue on https://github.com/cmdoret/dnaglider/issues/4#issue-1469927216 I am not sure when I will have time to make a conda package, but providing a dockerfile should be fairly simple (and better than nothing).
